site/en/docs/lighthouse/best-practices/paste-preventing-inputs/index.md

```diff
@@ -18,22 +18,23 @@ into password fields whenever users need to log in. This approach is generally
 more secure than forcing users to type in passwords that are short enough
 to remember.
 
+In the general case, any input not marked as readonly very likely should
+not disallow users from pasting into them.
```

Comment:
a little hard to parse this sentence. Maybe:

`In the general case, users shouldn't be prevented from pasting into any input field.`

and leave the "non-readonly" caveat for below?

Reply:
updated - good now?

Reply:
It's up to you. I can't articulate it, but to me "users shouldn't be prevented from pasting" seems more straightforward than "input elements should not disallow users from pasting".

I would also maybe drop the "very likely" part (the beginning already says we're discussing the general case), but again up to you :)